### DISCUSSION TITLE
Disable "embedded" rule

### DISCUSSION THREAD

**Discussion by chkpnt** [sveltejs/eslint-plugin-svelte#288] [Q&A] [answered]
I'm getting the following `svelte/valid-compile` error:

>  10:5  error  A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event.(a11y-click-events-have-key-events)  svelte/valid-compile

I do not want to disable `svelte/valid-compile` but only `a11y-click-events-have-key-events`. Is there a way to do this?

**Comment by chkpnt** [comment, marked as answer]
Okay, I have to use `svelte-ignore ...` instead of `eslint-disable-next-line ...`.

https://svelte.dev/docs#template-syntax-comments

**Reply by ota-meshi** [reply]
I think it's best to use `svelte-ignore` suppression comments as you said.
